test/test.py

```diff
@@ -284,6 +284,17 @@ def test_slicing(self):
         past_end = second_long_slice[:1500]
         self.assertTrue(second_long_slice._data == past_end._data)
 
+    def test_slicing_by_step(self):
+        audio = self.seg1[:10000]
+        chunks = audio[:0]
+
+        for chunk in audio[::1000]:
+            self.assertTrue(isinstance(chunk, audio.__class__))
```

Comment:
I think we can safely assert `isinstance(chunk, AudioSegment)`

Reply:
true!..I can wait to commit, to run Appveyor too

Reply:
ok Appveyor is fixed, just pull from pydub master